include/configs/ts7100.h

```diff
@@ -55,20 +55,7 @@
 	"scriptaddr=" __stringify(CONFIG_LOADADDR) "\0" \
 	"pxefile_addr_r=" __stringify(CONFIG_LOADADDR) "\0" \
 	"console=ttymxc0,115200\0" \
-	"chrg_pct=60\0" \
-	"chrg_verb=0\0" \
-	"silochargeon=silabs scaps disable;" \
-		"if test $silopresent = '1';" \
-			"then if test $jpnochrg = 'off';" \
-				"then silab scaps enable;"\
-			"fi;"\
-		"fi;\0" \
-	"silowaitcharge=if test $silopresent = '1';" \
-		"then if test $jpnochrg = 'on';" \
-			"then echo 'NO CHRG jumper is set, not waiting';" \
-			"else silabs scaps wait pct ${chrg_pct};" \
-		"fi;" \
-	"fi;\0" \
+	"silo_chrg_pct=60\0" \
```

Comment:
I'm not sure what the current paradigm is, but, with these changes is there:
1) A way to enable charging without having to set the `silo_chrg_pct` and wait for it?
2) Not have waiting for charge enabled out of the box so the first boot experience isn't the user waiting ~60 s for the platform to actually turn on.

Reply:
1) No, but a user could turn this on in linux with tsmicroctl. I'm not sure what the real use case is for this.
2) The TS-7100 does not have a no charge jumper.  During development the user can press ctrl+c to stop charging and boot.  I'll add a printf to make this more apparent that the user can do that to boot immediately.